.github/scripts/compile.sh

```diff
@@ -108,11 +144,11 @@ export CXXFLAGS="${CXXFLAGS} ${common_sanitizer_flags}"
 do_build "_asan" asan
 do_build "_tsan" tsan
 
-tar -z -C "$WORKING/proton_install" -cf /qpid-proton-image.tar.gz usr
+tar -z -C $PROTON_INSTALL_DIR -cf /qpid-proton-image.tar.gz usr
 
-DESTDIR=$WORKING/staging/ cmake --install $WORKING/build
-cp "$WORKING/build_asan/router/skrouterd" "$WORKING/staging/usr/sbin/skrouterd_asan"
-cp "$WORKING/build_tsan/router/skrouterd" "$WORKING/staging/usr/sbin/skrouterd_tsan"
-cp --target-directory="$WORKING/staging/" "$WORKING/tests/tsan.supp" "$WORKING/build_asan/tests/lsan.supp"
-tar -z -C "$WORKING/staging/" -cf /skupper-router-image.tar.gz usr etc lsan.supp tsan.supp
-#endregion
+DESTDIR=$SKUPPER_DIR/staging/ cmake --install $SKUPPER_BUILD_DIR
+cp "${SKUPPER_BUILD_DIR}_asan/router/skrouterd" "${SKUPPER_DIR}/staging/usr/sbin/skrouterd_asan"
+cp "${SKUPPER_BUILD_DIR}_tsan/router/skrouterd" "${SKUPPER_DIR}/staging/usr/sbin/skrouterd_tsan"
+cp --target-directory="${SKUPPER_DIR}/staging/" "${SKUPPER_DIR}/tests/tsan.supp" "${SKUPPER_BUILD_DIR}_asan/tests/lsan.supp"
+tar -z -C ""${SKUPPER_DIR}/staging/"" -cf /skupper-router-image.tar.gz usr etc lsan.supp tsan.supp
```

Comment:
```suggestion
tar -z -C "${SKUPPER_DIR}/staging/" -cf /skupper-router-image.tar.gz usr etc lsan.supp tsan.supp
```

typo, doubled quote sign